css/README.md

```diff
@@ -0,0 +1,85 @@
+# CSS
+An introduction to CSS in web programming. I have provided some resources and some practice code.
```

Comment:
Though these are great resources, they should not be phrased in such a personal manner, since this will represent MDN if your pull request is merged.